index.html

```diff
@@ -43,6 +44,19 @@
   <body class="w-full">
     <div id="app"></div>
     <script type="module" src="/src/main.js"></script>
-    <script defer data-domain="coracle.social" src="https://plausible.io/js/script.manual.js"></script>
+    <script
+      nonce="A132076C"
+      async
+      src="https://platform.twitter.com/widgets.js"
+      charset="utf-8"></script>
+    <script
+      nonce="A132076C"
+      async
+      src="https://platform.twitter.com/js/tweet.d7aeb21a88e025d2ea5f5431a103f586.js"
+      charset="utf-8"></script>
```

Comment:
I don't think we should do this, for privacy reasons. It basically makes it possible for twitter to track coracle users, even if the user has turned off link previews/embeds. Can we make this work by just adding it to the csp?

Reply:
Twitter only works this way, no iframe to pick the preview from another origin, it builds it on the fly via the widget library. That's ok the twitter preview from dufflepud is good enough.
Also removed Youtube, the client is blocking a bunch of request despite adding the origin in the frame-src, it seems like the iframe is doing a bunch of subsequent request that are all blocked from the origin. I sometimes get the proper preview but not always